src/Raven.Server/Config/Categories/ClusterConfiguration.cs

```diff
@@ -61,10 +61,11 @@ public class ClusterConfiguration : ConfigurationCategory
         [ConfigurationEntry("Cluster.TimeBeforeMovingToRehabInSec", ConfigurationEntryScope.ServerWideOnly)]
         public TimeSetting MoveToRehabGraceTime{ get; set; }
         
-        [Description("Tcp connection read/write timeout.")]
+        [Description("Tcp connection read/write timeout. (obsolete, use Server.Tcp.SendTimeout and Server.Tcp.ReceiveTimeout instead")]
         [DefaultValue(15 * 1000)]
         [TimeUnit(TimeUnit.Milliseconds)]
-        [ConfigurationEntry("Cluster.TcpTimeoutInMs", ConfigurationEntryScope.ServerWideOnly)]
+        [ConfigurationEntry("Server.Tcp.SendTimeout", ConfigurationEntryScope.ServerWideOrPerDatabase)]
+        [ConfigurationEntry("Cluster.TcpTimeoutInMs", ConfigurationEntryScope.ServerWideOrPerDatabase)] //changed to ServerWideOrPerDatabase to prevent incompatibility
         public TimeSetting TcpConnectionTimeout { get; set; }
```

Comment:
You removed all the occurences of this I think? What would be the reason to keep that configuration?

Reply:
Where is this used now?

Reply:
It is used by users, possibly.
Perhaps we should throw a "not supported" exception (with informative error) if it is set a value so users will know that this config value is no longer valid?

Reply:
Let's remove this configuration property entirely.